core/sodasql/cli/cli.py

```diff
@@ -62,6 +67,19 @@ def create(warehouse_type: str,
 
     WAREHOUSE_TYPE is one of {postgres, snowflake, redshift, bigquery, athena}
     """
+
+    soda_telemetry.set_attribute('cli_command_name', 'create')
```

Comment:
Just aside: it is annoying that we can't get the current function name within the function thanks to rejected https://www.python.org/dev/peps/pep-3130/